public/scripts/common/lib/handlebars.mixed.js

```diff
@@ -29,10 +29,53 @@ Displays select tag with options (and optional optgroups)
   * `selected`
 
 **/
+
 Handlebars.registerHelper('select', function(context, options) {
-  var output = `<select id="${options.hash.id}" name="${options.hash.name}" class="${options.hash.class}">`;
-  output += makeOptions(context, options);
-  return output +'</select>';
+  var addOption, addOptions,
+      output = '';
+
+  addOption = function(selected, _options) {
+    var {value, name, disabled, className, isSelected} = _options;
+    if(_.isNull(value)) value = 'null-' + _.uniqueId();
+    
+    if(_.isNull(isSelected)) {
+      isSelected = value == selected;
+    }
+
+
+    return  '<option' + 
+            (value ? ` value="${value}"` : '') +
+            (isSelected ? ' selected="selected"' : '') +
+            (disabled ? ' disabled="disabled"' : '') +
+            (className ? ` class="${className}"` : '') +
+            '>' + name + '</option>';
+  };
+
+  addOptions = function(selected, _options) {
+    return _.map(_options, _.partial(addOption, selected)).join('');
+  };
+
+  var requiredString = "";
+  if(options.hash.required) {
+    requiredString = " required ";
+  }
+
+  output += '<select id="' + options.hash.id + '" name="' +
+            options.hash.name + '" class="' + options.hash.class + '"' + requiredString + '>';
```

Comment:
@AJamesPhillips This should be triggering html5 validation, like the Short name.

It works as expected in Firefox, but on Chrome it only seems to be working when the 'Select RDP Part Type' string is also blank.

It's easy enough to 'fix' (don't have a string, or hack the string away on focus) so that the validation works, not sure if there's another way.